core/time/comparison_spec.rb

```diff
@@ -55,6 +55,22 @@
     }.should_not complain
   end
 
+  it "compares values using time zones" do
+    # three timezones, all at the same timestamp
+    time_utc = Time.new(2000, 1, 1, 0, 0, 0, 0)
+    time_cet = Time.new(2000, 1, 1, 1, 0, 0, '+01:00')
+    time_brt = Time.new(1999, 12, 31, 21, 0, 0, '-03:00')
+    (time_utc <=> time_cet).should == 0
+    (time_utc <=> time_brt).should == 0
+    (time_cet <=> time_brt).should == 0
+
+
+    # time_brt is later, even though the date is earlier
+    time_brt = Time.new(1999, 12, 31, 23, 0, 0, '-03:00')
+    (time_utc <=> time_brt).should == -1
+    (time_brt <=> time_utc).should == 1
+  end
+
```

Comment:
Taking into account how other tests in this file are fine granular - I think it makes sense to follow this approach and divide the new checks into separate test cases (under a new `context`, e.g. `given different timezones`). I would have separate cases for `1`, `-1` and `0` results.

Reply:
They're in a new context now. I think they would be more readable if we could just write `time_utc.should < time_brt`, but that feels out of place for the rest of this file.